### DISCUSSION TITLE
Any Possibility using Custom Port?

### DISCUSSION THREAD

**Discussion by jauhari** [EasyEngine/easyengine#1812] [Support Requests]
Hi we need help, we use CLOUDFLARED to make our **Local Server** Online on Public. The Problem is, if we want to make it _multiple site on same server_ we need to setup on Separate PORT on each Virtual Host

example 
✅ 80 (main site)
✅ 81 (other site)
✅ 82 (other other site)

We try to find on Easy Engine Support how to do that and didn't find the Right Solution. Any clues how to make it happen?

Regards

Easy Engine Maniac

**Comment by rodrigo-gpereira** [comment]
You don´t need to setup custom ports for each site, you can setup Cloudflared Instance with docker on same Local Server with Network Host (--network host), and setup all sites to 127.0.0.1 address in Cloudflare Tunnel Config, i´m using this setup for staging tests, and works fine.

<img width="931" height="397" alt="image" src="https://github.com/user-attachments/assets/44fb91bc-c358-4ac1-b4f6-3fd5c795ad3e" />
